src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double sideA;
+    private double sideB = sideA - 2;

```

Comment:
What is sideA - 2;? why this required?

Reply:
Try to avoid negative results.
Remove it.